### PR TITLE
Improve Listener + PHPCS

### PR DESCRIPTION
**Pull request by gperdomor**
Improve the listener to validate config parameters only when the lock is active.

Also i use the php-cs-fixer to fix code standards in the bundle

### CODE REVIEW

File: Listener/MaintenanceListener.php [hunk 1]
```diff
@@ -137,90 +135,80 @@ public function __construct(
...
         return;
     }
 
     /**
-     * Rewrites the http code of the response
-     *
-     * @param FilterResponseEvent $event FilterResponseEvent
-     * @return void
-     */
-    public function onKernelResponse(FilterResponseEvent $event)
```
Comment (spike31) [inline Listener/MaintenanceListener.php]: You can't remove that

File: Drivers/AbstractDriver.php [hunk 2]
```diff
@@ -33,28 +33,28 @@ public function __construct(array $options = array())
     }
 
     /**
-     * Test if object exists
+     * Test if is locked.
      *
      * @return boolean
      */
-    abstract public function isExists();
+    abstract public function isLocked();
```
Comment (spike31) [inline Drivers/AbstractDriver.php]: Don't rename this method

Comment (gperdomor) [inline Drivers/AbstractDriver.php]: done

File: Listener/MaintenanceListener.php (hunk 1 above)
Comment (gperdomor) [inline Listener/MaintenanceListener.php]: reverted

**Pull request closed without merging**